crates/storage_models/src/query/mandate.rs

```diff
@@ -32,12 +32,19 @@ impl Mandate {
         merchant_id: &str,
         customer_id: &str,
     ) -> StorageResult<Vec<Self>> {
-        generics::generic_filter::<<Self as HasTable>::Table, _, _>(
+        generics::generic_filter::<
+            <Self as HasTable>::Table,
+            _,
+            <<Self as HasTable>::Table as Table>::PrimaryKey,
+            _,
+        >(
             conn,
             dsl::merchant_id
                 .eq(merchant_id.to_owned())
                 .and(dsl::customer_id.eq(customer_id.to_owned())),
             None,
+            None,
+            None,
```

Comment:
If I pass `None` for the `ORDER BY` clause parameter (the generic type `O`) and add an underscore in the list of generic types, the compiler complains that it cannot infer the type of `O`. So, I've had to specify the primary key part as a dummy type for the `ORDER BY` clause.

Basically, I can't call it this way:

```rust
generics::generic_filter::<<Self as HasTable>::Table, _, _, _>(
                                                      // ^ This one can't be underscore
    conn,
    dsl::merchant_id
        .eq(merchant_id.to_owned())
        .and(dsl::customer_id.eq(customer_id.to_owned())),
    None,
    None,
    None, // Compiler can't infer type of this parameter
)
```